### PR TITLE
 Refine ctc model code for English dataset. 

### PR DESCRIPTION
**Pull request by wanghaoshuang**
1. Remove a pooling layer.
2. Change classes_num to 94.
3. Modify some arguments in ctc_train.py
4. Add learning rate decay policy.
5. Fix readme.

### CODE REVIEW

File: fluid/ocr_recognition/crnn_ctc_model.py [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+
+def poly_decay(learning_rate, total_step, power):
+    global_step = _decay_step_counter()
+    with init_on_cpu():
+        decayed_lr = learning_rate * (fluid.layers.pow(
+            (1 - global_step / total_step), power))
+    return decayed_lr
+
+
+def cosine_decay(learning_rate, total_step):
```
Comment (qingqing01) [inline fluid/ocr_recognition/crnn_ctc_model.py]: Remove `cosine_decay`?

File: fluid/ocr_recognition/ctc_train.py [hunk 2]
```diff
@@ -90,54 +90,57 @@ def train_one_batch(data):
...
+    def save_model(args, exe, iter_num):
+        filename = "model_%05d" % iter_num
         fluid.io.save_params(
             exe, dirname=args.save_model_dir, filename=filename)
         print "Saved model to: %s/%s." % (args.save_model_dir, filename)
 
-    for pass_id in range(args.pass_num):
-        batch_id = 1
+    iter_num = 0
+    while True:
```
Comment (qingqing01) [inline fluid/ocr_recognition/ctc_train.py:108]: Remove `while True`

**Review by qingqing01 — APPROVED** [review]

File: fluid/ocr_recognition/crnn_ctc_model.py (hunk 1 above)
Comment (wanghaoshuang) [reply fluid/ocr_recognition/crnn_ctc_model.py]: Fixed.
